src/database/migrations/20230824001544-add-indirection-domain.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn(
+      "launches", // name of Source model
+      "indirectionDomain", // name of column we're adding
```

Comment:
Hmm I was thinking it might be useful to have a table of mappings of primary domain -> indirection domain -> cloudfront domain, but I don't really have a strong opinion here. What are your thoughts?

Reply:
keeping it simple! unless we have a biz requirement where the hostedon domains differ!

Reply:
Sure! Removed in 0ca75981e751211247be85a810b785aee5bc3b7c.